### PR TITLE
add s01e02 normal themes

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/s01e02.json [hunk 1]
```diff
@@ -89,6 +89,62 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "collective WASH oriented activities",
+      "NumericValue": 8,
+      "StringValue": "collective_wash_oriented_activities",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-c518094e",
+      "CodeType": "Normal",
+      "DisplayText": "following right Covid19 prevention practise",
```
Comment (as2388) [inline code_schemes/s01e02.json]:
```suggestion
      "DisplayText": "following right covid19 prevention practice",
```

File: code_schemes/s01e02.json [hunk 2]
```diff
@@ -89,6 +89,62 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "duty holders providing adequate clean water",
+      "NumericValue": 10,
+      "StringValue": "duty_holders_providing_adequate_clean_water",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-2e5cb89e",
+      "CodeType": "Normal",
+      "DisplayText": "right wash practices",
```
Comment (as2388) [inline code_schemes/s01e02.json:119]: should this be wash or WASH?

File: code_schemes/s01e02.json [hunk 3]
```diff
@@ -89,6 +89,62 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "clean the environment",
+      "NumericValue": 13,
+      "StringValue": "clean_the_environment",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-d30af4db",
+      "CodeType": "Normal",
+      "DisplayText": "other",
```
Comment (as2388) [inline code_schemes/s01e02.json]:
```suggestion
      "DisplayText": "other theme",
```

**Review by as2388 — APPROVED** [review]

File: code_schemes/s01e02.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/s01e02.json:119]: yes, updated thanks.

File: code_schemes/s01e02.json (hunk 3 above)
Comment (IsaackMwenda) [reply code_schemes/s01e02.json]: updated thanks.

File: code_schemes/s01e02.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/s01e02.json]: updated thanks

**Pull request merged**